app/docs/chart-selection-tabs.docs.tsx

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable import/no-anonymous-default-export */
+import { markdown, ReactSpecimen } from "catalog";
+import React from "react";
+
+import { ChartSelectionTabs } from "@/components/chart-selection-tabs";
+import { ConfiguratorStateConfiguringChart } from "@/configurator";
+import { ConfiguratorStateProvider } from "@/src";
+import palmerPenguinsFixture from "@/test/__fixtures/int/scatterplot-palmer-penguins.json";
+
+export default () => markdown`
+> Chart Selection Tabs are used in ChartPreview and ChartPublished to enable selecting a chart type in any stage of creating a chart.
+
+They can be either _editable_, to display a button to show ChartTypeSelector (used when creating a chart) or not, to disable that functionality (used in published charts).
```

Comment:
Ah yes I was wondering what was the use of the editable prop.

Nit: It could be nice to have it as prop documentation

```
/** Tabs are not editable when they are published **/
editable: boolean
```

so that you get autocomplete documentation.

This is not important in this PR, just for the future, I think it's good to take the habit to document props when they are not obvious (I say this also for me ;)).